docs/poorsManDoc.rst

```diff
@@ -0,0 +1,11 @@
+You need to:
```

Comment:
Just put this in the README.rst file please (actually, create it with this content)